src/mlmc/synth_simulation.py

```diff
@@ -288,18 +288,4 @@ def _read_config():
         return config
 
```

Comment:
Should we have SyntSimulationn as part of the library or rather move it into tests?

Reply:
It should be here, I think there was some error around pickle if it wasn't in mlmc module.